pymongo/topology_description.py

```diff
@@ -361,7 +361,7 @@ def updated_topology_description(topology_description, server_description):
             topology_description._topology_settings)
 
     if topology_type == TOPOLOGY_TYPE.Unknown:
-        if server_type == SERVER_TYPE.Standalone:
+        if server_type in (SERVER_TYPE.Standalone, SERVER_TYPE.LoadBalancer):
```

Comment:
So with multiple seeds + LB we will now end up removing hosts until we have only one left and then connect to that as a Single?
Is this what the spec says to do or is this behavior undefined?

Reply:
See the comment. This is unreachable code in practice. It's just a failsafe in case the "serviceId" popping logic doesn't work or gets removed prematurely.